### ISSUE TITLE
exceptions when running under PS core 7.1.3 on mac

### ISSUE THREAD

**Issue by dsoper2** [CiscoDevNet/intersight-powershell#15]
A user reported an issue with PS core on mac and I duplicated the issue locally with exceptions being thrown on any cmdlets (example script from the powershell-utils repo):

Get-IntersightAaaAuditRecord: /Users/dsoper/Documents/intersight-powershell-utils/audit-examples/audit-user-summary.ps1:26:2
Line |
  26 |  (Get-IntersightAaaAuditRecord `
     |   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception has been thrown by the target of an invocation.

PS /Users/dsoper/Documents/intersight-powershell-utils> $psversiontable 

Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Darwin 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:33 …
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

Same API keys and PS core version does work for me on Windows server 2019

**Comment by Ghufz** [comment]
Hi David, 
The Get cmdlet exception is not clear due to some condition failure when target server is not reachable. 
I am working on it to fix it in c# SDK. 
just for checking try to create  new organization 
```powershell
$org = New-IntersightOrganizationOrganization -Name TestOrg
```
If exception occurs check for $Error[0]. Exception it will give detailed exception.

**Comment by Ghufz** [comment]
This issue has been resolved in the release 1.0.9.4430
